File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/main/java/simulacionBandaMusica/factories/TrompetistaFactory.java
package simulacionBandaMusica.factories;

import simulacionBandaMusica.models.Trompetista;

import static simulacionBandaMusica.utils.sorteoDatos.*;

public class TrompetistaFactory {

    public static Trompetista create() {
        String name = randomName();
        int experienceYear = randomAnno();
        int pulmonarCapacity = randomPulmonarCapacity();
        return new Trompetista(name, "Trompeta", experienceYear, pulmonarCapacity);
    }

}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/main/kotlin/simulacionBandaMusica/factories/TrompetistaFactory.kt
package simulacionBandaMusica.factories

import simulacionBandaMusica.models.Musico
import simulacionBandaMusica.utils.sorteoDatos

object TrompetistaFactory {
    fun create(): Musico.Trompetista {
        val name = sorteoDatos.randomName()
        val experienceYear = sorteoDatos.randomAnno()
        val pulmonarCapacity = sorteoDatos.randomPulmonarCapacity()
        return Musico.Trompetista(name, "Trompeta", experienceYear, pulmonarCapacity)
    }
}